resources/views/individuals/partials/progress.blade.php

```diff
@@ -31,6 +38,13 @@
                 <p id="cannot-publish-explanation">
                     {{ __('You must attend an orientation session and fill in all the required information before you can publish your page.') }}
                 </p>
+                @can('publish', $individual)
+                    <x-interpretation class="interpretation--start" name="{{ __('Publish page', [], 'en') }}"
+                        namespace="publish_page" />
+                @else
+                    <x-interpretation class="interpretation--start" name="{{ __('Publish page (not available)', [], 'en') }}"
+                        namespace="publish_page-disabled" />
+                @endcan
```

Comment:
This is nested in the `@cannont` so the `<x-interpretation>` component for the publish button is never reachable. Rather than having a `@can` directive here the publish button interpretation should be moved up to where the publish button is actually added in the template on line 33. The interpretation for the when the it isn't publishable can stay where it is.